.github/workflows/prepare-release.yaml

```
@@ -0,0 +1,41 @@
+name: Prepare Release PR
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version to publish, e.g. 0.0.1'
+        required: true
+        default: '0.0.1'
+        type: string
+
+jobs:
+  prepare_release:
+    name: Prepare release PR
```

Nit: Maybe we use the same name as for the workflow? Just for consistency.
```suggestion
    name: Prepare Release PR
```